proposals/handler_test.go

```diff
@@ -806,7 +807,9 @@ func TestBallot_DecodeBeforeVotesConsistency(t *testing.T) {
 
 	decoded := &tortoise.DecodedBallot{BallotTortoiseData: b.ToTortoiseData()}
 	th.md.EXPECT().DecodeBallot(decoded.BallotTortoiseData).Return(decoded, expected)
-	require.ErrorIs(t, th.HandleSyncedBallot(context.Background(), b.ID().AsHash32(), peer, data), expected)
+	err := th.HandleSyncedBallot(context.Background(), b.ID().AsHash32(), peer, data)
+	require.ErrorIs(t, err, fetch.ErrIgnore)
+	require.Contains(t, err.Error(), expected.Error())
```

Comment:
With the suggested change above this becomes:

```suggestion
	require.ErrorIs(t, err, expected)
```